bpf/tests/hairpin_sctp_flow.c

```diff
@@ -28,6 +28,7 @@ mock_ctx_redirect_peer(const struct __sk_buff *ctx __maybe_unused, int ifindex _
 
 /* Set the LXC source address to be the address of pod one */
 ASSIGN_CONFIG(__u32, endpoint_ipv4, v4_pod_one)
+ASSIGN_CONFIG(__u32, ipv4_loopback, 0x1ffff50a)
```

Comment:
Be skeptical of the dummies specified in `node_config.h`, they were mostly placeholders to keep code from getting optimized out. Is this strictly necessary here? Do the tests pass without it? Could we use a non-magic value here instead? (e.g. v4_node_one)

Reply:
Tests don't pass without this. Before, they were getting the value from the defaults defined in `node_config.h` I believe. I can define `v4_loopback` or some such macro instead here.